release-tool

```diff
@@ -573,8 +578,16 @@ build() {
 
     OUTPUT_DIR="$(realpath "$OUTPUT_DIR")"
 
-    logInfo "Checking out release tag '${TAG_NAME}'..."
-    git checkout "$TAG_NAME"
+    if $BUILD_NIGHTLY; then
+        TAG_NAME="HEAD"
+        local branch=`git rev-parse --abbrev-ref HEAD`
+        logInfo "Using current branch ${branch} to build..."
+        local git_rev=`git rev-parse --short HEAD`
+        RELEASE_NAME="${RELEASE_NAME}_${git_rev}"
```

Comment:
Do we really want to add the git revision? It makes for a rather random sort order when you have multiple versions. I would also guess it's rare that you want to keep more than the latest snapshot, so it would be useful if downloading a new version had the same name as the old one, so it just replaces the file.

Reply:
Corrected in last push